src/Reshuffle.ts

```diff
@@ -74,7 +78,7 @@ export default class Reshuffle {
     }
   }
 
-  when(eventConfiguration: EventConfiguration, handler: (() => void) | Handler): Reshuffle {
+  when(eventConfiguration: EventConfiguration, handler: Handler | HandlerWrapper) {
```

Comment:
do we have one connector using the HandlerWrapper? if no then we should just delete that type and the check i the code

Reply:
I don't think so, just a test case. Let's remove it